yt/data_objects/static_output.py

```diff
@@ -215,9 +216,7 @@ def __init__(
         self.known_filters = self.known_filters or {}
         self.particle_unions = self.particle_unions or {}
         self.field_units = self.field_units or {}
-        if units_override is None:
-            units_override = {}
-        self.units_override = units_override
+        self.units_override = self.__class__._sanitize_units_override(units_override)
```

Comment:
Wow, I've never seen this before.

Reply:
what ? calling a class method from an instance ? I could be convinced this is an anti-pattern actually, I just don't know why ;)